app/desktop/src/Application.scss

```diff
@@ -0,0 +1,92 @@
+// $deep-blue: #32404e;
```

Comment:
@unnikrishnan-narayanan  Do we need all these comments?  Or can they just be deleted?

Reply:
These are coming in via the ext-gen app as example I think. for now, I'll leave as is